debian/py3dist-overrides

```diff
@@ -10,3 +10,6 @@ python_sonic python3-sonic
 pyv4l2camera python3-pyv4l2camera
 scipy python3-scipy
 simple_pid python3-simple-pid
+dlib python3-dlib
+scikit-learn python3-sklearn
```

Comment:
What is this for?

Reply:
It's to run an SVC for emotion classification - I figured we'd want it on the OS anyway since we plan on doing a lot more AI related content

Reply:
I think jorge added this after we discussed the emotion classification code, which isn't actually a part of this PR. I'll clean up the two branches and make the scope clearer